Integrations/integration-RsaNetwitnessSecurityAnalytics.yml

```diff
@@ -1081,57 +1297,60 @@ script:
         case 'nw-login':
             results = sessionId;
             break;
-        case 'nw-list-incidents':
+        case 'netwitness-im-list-incidents':
             var incidents = listIncidents(sessionId, args, incidentManagementId);
-            results = { incidents: incidents };
+            results = buildRetValListIncidents(incidents);
             break;
-        case 'nw-get-incident-details':
+        case 'netwitness-im-get-incident-details':
             var incident = getIncidentById(sessionId, args, incidentManagementId);
-            results = { incident: incident };
+            results = buildRetValGetIncidentDetails(incident);
             break;
-        case 'nw-get-components':
+        case 'netwitness-im-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = buildRetValGetComponents(components);
             break;
-        case 'nw-get-events':
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
             break;
-        case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+        case 'netwitness-im-get-event-details':
+            event = getEventDetails(sessionId, args);
+            results = buildRetValGetEventDetails(event);
             break;
-        case 'nw-get-alerts':
+        case 'netwitness-im-get-alerts':
             var alerts = filterAlerts(sessionId, args);
-            results = { alerts: alerts };
+            results = buildRetValGetAlerts(alerts);
             break;
-        case 'nw-get-alert-details':
+        case 'netwitness-im-get-alert-details':
             var alert = getAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlert(alert);
             break;
-        case 'nw-get-alert-original':
+        case 'netwitness-im-get-alert-original':
             var alert = getOriginalAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlertOrig(alert);
```

Comment:
The same with `==` happening here
![image](https://user-images.githubusercontent.com/7270217/28249416-e68118e2-6a5d-11e7-9e8e-0e795a4e7027.png)
![image](https://user-images.githubusercontent.com/7270217/28249419-f286ab52-6a5d-11e7-9ba3-cf064ec427ed.png)